src/dynlib/dynlib_linux.c

```diff
@@ -11,6 +11,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <ctype.h>
+#define __USE_GNU
```

Comment:
I think we want to use `_GNU_SOURCE`

https://stackoverflow.com/questions/7296963/gnu-source-and-use-gnu

```suggestion
#define _GNU_SOURCE
```